### ISSUE TITLE
Isomorphic (Universal) feature

### ISSUE THREAD

**Issue by kikoanis** [hung-phan/koa-react-isomorphic#6]
Not sure if this is a bug or just overlooked behaviour. My understanding is that one of the reasons for isomorphic (Universal) applications is for screens to be served from server and clients even if javascript is disabled on browsers. I know the user might see a screen that is not functional as javascript is disabled and most of the functionality involves javascript (most of the time).
As of now if you disable javascript you won't see anything on any screen. My thinking is because of using jquery on document ready (I might be wrong). 

So the question is, is this a valid and intended behaviour?

**Comment by hung-phan** [comment]
What is the env you are running on? development or production? If it is dev then try.

```
$ SERVER_RENDERING=true npm run watch
$ npm run dev
```

**Comment by hung-phan** [comment]
You will see sth like this when you disable the js.

<img width="1280" alt="screen shot 2015-09-01 at 10 42 19 pm" src="https://cloud.githubusercontent.com/assets/3093244/9623809/d247dcd6-50fa-11e5-8479-5b5f84285750.png">

**Comment by hung-phan** [comment]
In production mode, you will get the styles by running

```
$ gulp build
$ SECRET_KEY=your_env_key npm start
```

**Comment by kikoanis** [comment]
O.K. I think I found the issue. If you run in dev mode (watch and then run dev) and then you disable javascript, you get a blank screen. I guess this is related to how the watch process works. Then even if you stop the 'watch' process, and restart dev process, you still get a blank screen. 

Anyway, I should have tested the whole thing fully before I opened an issue. 

Thank you for maintaining the project. Keep up the good work
